source/upgrade.txt

```diff
@@ -151,6 +151,24 @@ Version 5.0 Breaking Changes
   responding in the same way as ``Decimal128`` values. In particular,
   ``BsonDecimal128.isNumber()`` now returns ``true``, and
   ``BsonDecimal128.asNumber()`` returns the equivalent ``BsonNumber``.
+  
+- This driver version removes the `ServerAddress <{+api+}/apidocs/mongodb-driver-core/com/mongodb/ServerAddress.html>`__
+  methods ``getSocketAddress()`` and ``getSocketAddresses()``.
+
+  Instead of ``getSocketAddress()``, use the ``getByName()`` instance
+  method of ``java.net.InetAddress``.
+
+  Instead of ``getSocketAddresses()``, use the ``getAllByName()`` instance
+  method of ``java.net.InetAddress``.
+
+- This driver version removes the `UnixServerAddress <{+api+}/apidocs/mongodb-driver-core/com/mongodb/UnixServerAddress.html>`__
+  methods ``getSocketAddress()`` and ``getUnixSocketAddress()``.
+
+  Instead of ``getSocketAddress()`` or ``getUnixSocketAddress()``, construct an instance of
```

Comment:
There is no replacement for `UnixServerAddress.getSocketAddress`. Note also that `UnixServerAddress.getSocketAddress` has always been throwing `UnsupportedOperationException`.

Reply:
I didn't add anything about the `UnsupportedOperationException` since we're removing it.